NAMESPACE

```diff
@@ -4,3 +4,5 @@ export(add_two_numbers)
 export(download_from_azure_blob)
 export(fetch_blob_container)
 export(fetch_credential_from_env_var)
+export(read_interval_pmf)
+export(read_parameters)
```

Comment:
`read_parameters_from_x`? For some choice of `x`? `read_parameters` is quite general

Reply:
I'm not really following...

What are you trying to differentiate between here? I think `read_parameters_from_here()` implies there's also a `read_parameters_from_there()`. The added specificity in the name evokes the contrast between the two.

 I was going for a short, descriptive name that made clear we were reading the `parameters` (as opposed to the `data`). Is there some other range of choice you think need to be made clear in the function name?

Reply:
The possible range of choices are 1) which parameters and 2) from which type of file. For example just above there is this function `fetch_credential_from_env_var` which has both what it is getting and from where. To me having not really worked on this codebase something like `read_parameters` is quite short.

I guess part of the challenge is that it's a wrapper around `read_interval_pmf` which is more descriptive right. To illustrate why I think it's slightly weird, `read_parameters` only calls different versions of `read_interval_pmf` and yet the information that it's reading interval PMFs is now dropped from the name in `read_parameters`.

So what about something like `read_all_interval_pmf`. Or some other way to say this is basically a list of three calls to `read_interval_pmf`.

(I'm withdrawing interest in `from_` -- I think the key thing for me is what parameters are we reading.)

Reply:
Are we planning to extend this function beyond intervals PMFs? The title is like "disease process parameters" yet these are all interval PMFs

Reply:
Another option could be something like `read_disease_parameters`

Reply:
I like `read_disease_parameters`!